src/services/resource-service.ts

```diff
@@ -122,6 +123,15 @@ export const ResourceService = {
       })
     })
   },
+  getFinishedQuizzesByQuizId: async (cooperationId: string, quizId: string) => {
```

Comment:
It's not async, remove that please
```suggestion
  getFinishedQuizzesByQuizId: (cooperationId: string, quizId: string) => {
```